src/core/lib/iomgr/timer_custom.cc

```diff
@@ -24,15 +24,13 @@
 #include <grpc/support/log.h>
 
 #include "src/core/lib/debug/trace.h"
-#include "src/core/lib/iomgr/iomgr_custom.h"
 #include "src/core/lib/iomgr/port.h"
 #include "src/core/lib/iomgr/timer.h"
 
 static grpc_custom_timer_vtable* custom_timer_impl;
 
 void grpc_custom_timer_callback(grpc_custom_timer* t,
                                 grpc_error_handle /*error*/) {
-  GRPC_CUSTOM_IOMGR_ASSERT_SAME_THREAD();
```

Comment:
I believe this was required for the gevent implementation, so this can be merged after the gevent migration lands.